jerry-core/vm/vm.c

```diff
@@ -184,7 +184,7 @@ vm_op_set_value (ecma_value_t object, /**< base object */
 /**
  * Decode table for both opcodes and extended opcodes.
  */
-static const uint16_t vm_decode_table[] =
+static const uint16_t vm_decode_table[] STORE_IN_ROM =
```

Comment:
short table[4] = STORE_IN_ROM { 1,2,3,4 }
    in main() ->
      printf("%d\n", (int)sizeof(table)) // eight or sixteen?